go.mod

```diff
@@ -119,3 +138,11 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
+
+replace (
+	k8s.io/api => k8s.io/api v0.31.1
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.31.1
+	k8s.io/apimachinery => k8s.io/apimachinery v0.31.1
+	k8s.io/client-go => k8s.io/client-go v0.31.1
+	k8s.io/component-base => k8s.io/component-base v0.31.1
+)
```

Comment:
Can we please move this to the top of the file so it's clear these are replaced?